src/truststore/__init__.py

```diff
@@ -1,13 +1,34 @@
 """Verify certificates using native system trust stores"""
 
+import ssl
```

Comment:
I don't understand why we need this import both here and on line 12?

Reply:
This was added by my IDE, apologies for that.